ui/src/router.tsx

```diff
@@ -22,29 +22,19 @@ import { EditProject } from './routes/project/edit-project';
 
 const root = path('/');
 const project = root.path('/project');
-const inference = root.path('/inference');
-const dataset = root.path('/dataset');
-const models = root.path('/models');
-const labels = root.path('/labels');
+const inference = project.path('/:projectId/inference');
```

Comment:
General suggestion: make the route be `/projects` and not `/project`, then you can use a `project` path here and down below

```suggestion
const projects = root.path('/projects');
const project = projects.path('/:projectId');
const inference = project.path('/:projectId/inference');
```

Reply:
Agree. Updated